client/components/container/container-directive.css

```diff
@@ -1,17 +1,27 @@
-.pk-container {
-  border: 1px solid #fff;
+.pk-container-content {
+  background: #f5f5f5;
+  display: flex;
+  display: -webkit-flex;
+  padding: 2px;
 }
 
-.pk-container-selected {
-  background: #F1F1F1;
-  border: 1px dashed;
-  border-color: #D6D6D6;
-  border-radius: 4px;
+.pk-container:not(.pk-container-selected) > .pk-container-content:hover,
+.pk-container:not(.pk-container-selected-partial) > .pk-container-content:hover {
+  background: #e0e0e0;
+  border: 1px dashed #bdbdbd;
+  padding: 1px;
 }
 
-.pk-container-content {
-  display: flex;
-  display: -webkit-flex;
+.pk-container-selected > .pk-container-content {
+  background: #e0e0e0;
+  border: 2px solid #616161 !important;
```

Comment:
Optional: for !important, it can be helpful to add a comment that points to the style being forcibly overriden and/or an explanation why it's needed. Or alternatively a block comment for a set of related styles to explain how they interact. In this case I think it would be helpful to note the assumption that the sum of border + padding sizes must remain constant to avoid layout yank (assuming I understand it right).